applications/vessel-history/src/components/Splash.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { memo } from 'react'
+import cx from 'classnames'
+import Logo from '@globalfishingwatch/ui-components/dist/logo'
+import vesselHistoryLogo from '../assets/images/splash-screen-image@2x.png'
+import tmtLogo from '../assets/images/tmt_logo_final_full_colour@2x.png'
+import styles from './Splash.module.css'
+
+function Splash() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.splash}>
+        <img src={vesselHistoryLogo} className={styles.vhLogo} alt="Vessel History" />
```

Comment:
The name `Vessel History` is not set we need to remember all places where we are stating the name in case we need to change it.

Reply:
Once I18N is in place later in this sprint (https://globalfishingwatch.atlassian.net/browse/VH-54) we'll be able to store all strings in the `translations.json` files, makes sense?

Reply:
makes sense for me